rust-runtime/aws-smithy-http-server-python/examples/pokemon_service.py

```diff
@@ -195,6 +200,18 @@ def do_nothing(_: DoNothingInput) -> DoNothingOutput:
 def get_pokemon_species(
     input: GetPokemonSpeciesInput, context: Context
 ) -> GetPokemonSpeciesOutput:
+    if context.lambda_ctx is not None:
+        logging.debug(
+            "Lambda Context: %s",
+            dict(
```

Comment:
It's a shame there's no interface on the `logging` package to do structured logging, just serializing this big dict seems wrong.

Reply:
There is, but it is something the customer should provide through their own logging format (I have used a lot JSON lines in the past).

So to be as compatible as possible, we just log the entire dict.